### PR TITLE
created ozlem class

### PR DESCRIPTION
**Pull request by ocetin35**

### CODE REVIEW

File: src/week1/ozlem.java [hunk 1]
```diff
@@ -0,0 +1,8 @@
+package week1;
+
+public class ozlem {
+    public static void main(String[] args) {
+
+        System.out.println("Hello this is ozlem");
```
Comment (ocetin35) [inline src/week1/ozlem.java]: please change this line :) please remove the "hello" word.

**Comment by hsnakbba** [comment]
i am here :)

File: src/week1/ozlem.java (hunk 1 above)
Comment (ocetin35) [reply src/week1/ozlem.java]: its done. Please verify.

Comment (ocetin35) [reply src/week1/ozlem.java]: this has been resolved.

**Review by rmautia — APPROVED** [review]

**Review by hsnakbba — APPROVED** [review]

**Pull request merged**